src/pages/study-room/components/participant-list-item.test.tsx

```diff
@@ -0,0 +1,16 @@
+import { screen } from '@testing-library/react';
+import { render } from '@/test-helper';
+import fixtures from '@/fixtures';
+import ParticipantListItem from './participant-list-item';
+
+const { participants } = fixtures;
+
+const participant = participants[0];
```

Comment:
보통 배열에 인덱스로 접근하는 것 자체를 매직넘버이기도 하고 지양해서 아래와 같이 표현해주는편인 것 같아요~
```suggestion
const [participant] = participants;
```

Reply:
오오 이렇게 작성할수도 있군요 감사합니다!